src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/RazorCompletionItemProvider.cs

```diff
@@ -4,12 +4,11 @@
 #nullable disable
 
 using System.Collections.Generic;
-using Microsoft.AspNetCore.Razor.Language;
 
 namespace Microsoft.CodeAnalysis.Razor.Completion
 {
     internal abstract class RazorCompletionItemProvider
     {
-        public abstract IReadOnlyList<RazorCompletionItem> GetCompletionItems(RazorCompletionContext context, SourceSpan location);
+        public abstract IReadOnlyList<RazorCompletionItem> GetCompletionItems(RazorCompletionContext context);
```

Comment:
We only ever used the absolute index of source span so baked that concept into the completion context and baked the "owner" into it.